bundler/src/main/resources/log4j2.yml

```diff
@@ -18,10 +18,27 @@ Configuration:
       follow: true
       PatternLayout:
         pattern: "${LOG_PATTERN}"
+    RollingFile:
+      name: File
+      fileName: ${logDir}/bundler.log
+      filePattern: "${logDir}/bundler.log.%d{yyyy-MM-dd-hh}"
+      Policies:
+        TimeBasedTriggeringPolicy:
+          interval: 1
+          modulate: true
+      DefaultRolloverStrategy:
+        Delete:
+          basePath: ${logDir}
+          IfLastModified:
+            age: 5d
+      PatternLayout:
+        pattern: "%d{yyyy-MM-dd HH:mm:ss.SSS} %5p %c{3} -- %m%n"
```

Comment:
Simplified the pattern since colors or control characters are not needed for log aggregation.
Also not logging PID or thread. Don't think we need those but can add them if other's think we should